shell/browser/electron_browser_client.cc

```diff
@@ -872,6 +872,14 @@ bool ElectronBrowserClient::ShouldUseProcessPerSite(
 #endif
 }
 
+bool ElectronBrowserClient::ArePersistentMediaDeviceIDsAllowed(
+    content::BrowserContext* browser_context,
+    const GURL& scope,
+    const GURL& site_for_cookies,
+    const base::Optional<url::Origin>& top_frame_origin) {
+  return true;
```

Comment:
Might be better to return `false` for `browser_context->IsOffTheRecord()` types, I don't see upstream covering that condition.

Reply:
Nm: I misunderstood the persistent level of id , the spec makes it clear https://w3c.github.io/mediacapture-main/#privacy-and-security-considerations